File: src/main/java/com/market/ddanggyeo/account/entity/Account.java
package com.market.ddanggyeo.account.entity;

import com.market.ddanggyeo.utils.BaseTimeEntity;
import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;

@Getter
@Setter
@Entity
public class Account extends BaseTimeEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    @Column(nullable = false, unique = true)
    private String email;
    @Column(nullable = false)
    private String password;
}

File: src/main/kotlin/com/market/ddanggyeo/account/entity/Account.kt
package com.market.ddanggyeo.account.entity

import com.market.ddanggyeo.utils.BaseTimeEntity
import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType
import javax.persistence.Id

@Entity
class Account(email: String, password: String): BaseTimeEntity() {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long? = null

    @Column(nullable = false, unique = true)
    var email: String = email

    @Column(nullable = false)
    var password: String = password
}